grpc-interface-support/test/integration/test_integration_python.py

```diff
@@ -92,7 +95,30 @@ async def wait_until_ready(self):
         async def stop(self):
             pass
 
-    middleware = TestMiddleware()
+    print("============= BUILDING SERVER ===================")
+    os.chdir(os.environ["SERVICE_SERVER_ROOT"])
+    assert subprocess.check_call(["velocitas", "init", "-v"]) == 0
+
+    from seats_service_sdk.seats_pb2_grpc import SeatsServicer
```

Comment:
please also add tests for the local service generation, in the testfolder is a horn.proto which should generate a horn-service